src/components/VerifyPageContent/helpers.ts

```diff
@@ -0,0 +1,14 @@
+import { NETWORK_NAME } from "../../config";
+
+type LoadCertificate = (certificate: any) => void;
+
```

Comment:
`certificate` should we type it?

Reply:
eventually, i'll need to look into the types itself, i dont really understand that part yet

Reply:
i guess can `WrappedDocument<v2.OpenAttestationDocument>` first? later on when v3 need to update again i think